packages/valory/connections/p2p_libp2p/tests/test_aea_cli.py

```diff
@@ -93,47 +93,15 @@ def run_aea_cli_test(self) -> None:
         assert self.is_successfully_terminated(process)
 
 
-@libp2p_log_on_failure_all
```

Comment:
Moved to packages/valory/connections/test_libp2p/tests/test_p2p_libp2p/test_aea_cli.py